explainer.md

```diff
@@ -71,56 +64,32 @@ for all new messages including group chats not directly addressed to the user.)
 
 There is a single global badge associated with each Web
 application (as defined in [Web app
-manifest](https://www.w3.org/TR/appmanifest/)). At any time, the badge is set to
-either:
-
-* Nothing (the badge is "cleared"), or
-* A "flag" indicating the presence of a badge with no contents, or
-* A non-empty string containing a single *[grapheme
-  cluster](http://unicode.org/reports/tr29/#Grapheme_Cluster_Boundaries)*
-  (roughly: a single Unicode character), or
-* A positive integer.
-
-The model does not allow a badge that is the empty string, a negative integer,
-or the integer value 0.
+manifest](https://www.w3.org/TR/appmanifest/)). At any time, the badge is set with:
 
-The grapheme cluster allows a single character to be represented, but also
-for combining characters to be combined into what the user thinks of as a single
-character. Examples of grapheme clusters include:
-
-* "a" (a single character, Latin "a")
-* "நி" (a base character + combining character, Tamil "ni")
-* 👩🏾‍🔬️ (emoji combination with skin tone modifier, Woman Scientist with
-  Medium-Dark Skin Tone)
+* Nothing (we display a notification "flag"), or
+* A positive integer.
 
-Clients are encouraged to use integers instead of digit characters, because some
-operating systems may *only* support integer badges, so using the digit
-character would not display correctly. Also integers can be localized better for
-the user.
+The model does not allow a badge that is a negative integer or the integer value 0.
 
 ### The API
 
 The `Badge` interface is a member object on
-[`Window`](https://html.spec.whatwg.org/#the-window-object) and
-[`Worker`](https://www.w3.org/TR/workers/#worker). It contains two methods:
+[`Window`](https://html.spec.whatwg.org/#the-window-object)
 
-* `void set(optional USVString or long)`: Sets the associated app's badge to the
+* `void set(optional long)`: Sets the associated app's badge to the
   given data, or just "flag" if the argument is not given.
 * `void clear()`: Sets the associated app's badge to nothing.
 
-These can be called from either a foreground page or a service worker (in either
-case, affecting the whole app, not just the current page).
+These can be called from a foreground page 
```

Comment:
"... from a foreground page only (calling from a service worker is planned for future)."